packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
 
 import 'dart:async';
 import 'dart:collection';
```

Comment:
ObjectKey.value should probably be nullable

Reply:
Done